src/Sfa.Tl.Matching.Web/Controllers/ProviderController.cs

```diff
@@ -39,7 +39,14 @@ public IActionResult Start()
         [Route("find-providers", Name = "Providers_Get")]
         public IActionResult Index()
         {
-            return GetIndexViewAsync();
+            var viewModel = new SearchParametersViewModel
+            {
+                SelectedRouteId = null,
+                Postcode = null,
+                SearchRadius = SearchParametersViewModel.DefaultSearchRadius
+            };
```

Comment:
why this change?

Reply:
I made this refactoring because the number of parameters of the method was growing and too many primitive data types were being used.